app/models/job.rb

```diff
@@ -1,5 +1,5 @@
 class Job < ApplicationRecord
-  include_concern 'StateMachine'
+  include StateMachineMixin
```

Comment:
Looks like we can have a separate PR to introduce `StateMachineMixin` and refactor `Job`.

Reply:
@hsong-rh I agree with @bzwei we should have separate PRs.
I would have one for maybe the rubicon warnings
One for refactoring from Job
Then have one for the V2V changes

Reply:
Ok, I will split this into two PRs: this one will keep for v2v, the other for statemachine mixin refactor for job/request.